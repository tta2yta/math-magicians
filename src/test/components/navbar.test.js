import React from 'react';
import renderer from 'react-test-renderer';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import { BrowserRouter as Router } from 'react-router-dom';
import Navbar from '../../components/NavBar';

let getByTestId;
let component;

beforeEach(() => {
  component = render(
    <Router>
      <Navbar />
    </Router>,
  );

  getByTestId = component.getByTestId;
});

it('renders the Navbar component', () => {
    component = renderer
      .create(
        <Router>
          <Navbar />
        </Router>,
      )
      .toJSON();
  
    expect(component).toMatchSnapshot();
  });
