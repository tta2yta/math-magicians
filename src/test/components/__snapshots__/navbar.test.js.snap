// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Navbar component 1`] = `
<div
  className="navBar"
>
  <h1
    data-testid="heading"
  >
    Math Magicians
  </h1>
  <ul>
    <a
      aria-current="page"
      className="active-link"
      href="/"
      onClick={[Function]}
      style={Object {}}
    >
      Home
    </a>
    <a
      aria-current={null}
      href="/calculator"
      onClick={[Function]}
    >
      Calculator
    </a>
    <a
      aria-current={null}
      href="/Qoutes"
      onClick={[Function]}
    >
      Qoute
    </a>
  </ul>
</div>
`;
